src/pushsource/_impl/schema/staged-schema.yaml

```diff
@@ -212,6 +216,20 @@ definitions:
         - "null"
         minLength: 1
 
+    ami_billing_codes:
+        # Billing codes for an AMI
+        type:
+        - object
+        - "null"
+        properties:
+            name:
+                type: string
+            codes:
+                type: array
```

Comment:
I think this will allow an array of any type of object, can we please make it an array of string?